WooCommerce/WooCommerceTests/Tools/MockOrders.swift

```diff
@@ -70,7 +75,7 @@ final class MockOrders {
                      coupons: [],
                      refunds: refunds,
                      fees: fees,
-                     taxes: [] // TODO: 5809 - Add sampleOrderTaxLines method
+                     taxes: [tax]
```

Comment:
Similar comment to the one on the `ReceiptStoreTests` file - I think we can just simply receive `taxes` as input for the `makeOrder` method for simplicity.

Reply:
Done in - https://github.com/woocommerce/woocommerce-ios/pull/5931/commits/561c6f19f9044da4fb8f91796917cf455adab354